dags/bqetl_urlbar.py

```diff
@@ -48,27 +48,9 @@
     doc_md=docs,
     tags=tags,
 ) as dag:
-    firefox_desktop_urlbar_events__v1 = bigquery_etl_query(
-        task_id="firefox_desktop_urlbar_events__v1",
-        destination_table="urlbar_events_v1",
-        dataset_id="firefox_desktop_derived",
-        project_id="moz-fx-data-shared-prod",
-        owner="akommasani@mozilla.com",
-        email=[
-            "akomar@mozilla.com",
-            "akommasani@mozilla.com",
-            "anicholson@mozilla.com",
-            "dzeber@mozilla.com",
-            "rburwei@mozilla.com",
-            "telemetry-alerts@mozilla.com",
-        ],
-        date_partition_parameter="submission_date",
-        depends_on_past=False,
-    )
-
     firefox_desktop_urlbar_events__v2 = bigquery_etl_query(
         task_id="firefox_desktop_urlbar_events__v2",
-        destination_table="urlbar_events_v2",
+        destination_table="urlbar_events_v1",
```

Comment:
Is this supposed to build v1?

Reply:
we are bumping the version of the table.going forward it will e `v2`

Reply:
Fixed it. Thank you for point that out